SDK_dotnet/Clients/GluwaClient.cs

```diff
@@ -323,7 +323,7 @@ public GluwaClient(Environment env)
             {
                 if (nonce == null)
                 {
-                    nonce = ((int)(DateTime.UtcNow - new DateTime(1970, 1, 1)).TotalSeconds).ToString();
+                    nonce = nonce = GluwaService.GetNonceString();
```

Comment:
`nonce = nonce = GluwaService.GetNonceString();`

is the middle `nonce =` necessary?